src/jquery/data.js

```diff
@@ -38,3 +39,39 @@ jQuery.data = function( elem, name, value ) {
 
 	return oldData.apply( this, arguments );
 };
+
+jQuery.fn.extend( {
+    data: function( key, value ) {
+        var args, result;
+        if ( arguments.length === 0 && typeof Proxy !== "undefined" ) {
+            result = oldFnData.call( this );
+            // eslint-disable-next-line no-undef
```

Comment:
Why is it needed when all variables are used?

Reply:
I got an eslint error that Proxy is not defined, could it be only for me?

Reply:
Ah, this. Instead of silencing the whole rule, please add another override entry to `src/.eslintrc.json`. There's already one for `css.js`, you can copy it and just include `Proxy` as we don't use `Reflect` here.